nx.json

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "namedInputs": {
+    "default": [
```

Comment:
What is the rationale to adjust this?

Reply:
I think these default inputs make more sense in our case than the generated defaults.
I mean, why should we by default exactly exclude the jest config and nothing else?
Also, ATM, we don't have any shared inputs, probably this changes once we have to pass the Nx cloud token. 

<img width="1075" height="522" alt="image" src="https://github.com/user-attachments/assets/c4f10250-6b93-40c7-a91d-a6acf7abb8ef" />

https://nx.dev/reference/inputs#named-input-conventions

AFAIU we don't have take the generated inputs as they are. (BTW, I think I remember they where anyway differently generated than the ones in the screenshot, because we had a Turborepo before).

Does something speak against these defaults?

Reply:
I am mostly worried that changing inputs in the defaults leads to scenarios like with a dockerignore where suddenly something is missing in the context, and it is hard to debug why that happens. So I think we should tweak the default as little as possible and It did not really get through to me what the benefit is to exclude the docker configs